doc/classes/Viewport.xml

```diff
@@ -116,6 +116,13 @@
 				Returns the visible rectangle in global screen coordinates.
 			</description>
 		</method>
+		<method name="gui_find_control">
+			<return type="Control" />
+			<param index="0" name="point" type="Vector2" />
+			<description>
+				Returns the top-level [Control] at the given point within this viewport. If no [Control] is at the point, returns null.
```

Comment:
The wording "top-level Control" is ambiguous in this case and should be avoided.
For Control nodes there is the CanvasItem property `top_level` which could be confused.
See https://docs.godotengine.org/en/latest/classes/class_canvasitem.html#class-canvasitem-property-top-level

Reply:
Thanks for your help on reviewing this. I committed the suggested changes from AThousandShips down below.